src/i_system.c

```diff
@@ -49,7 +49,11 @@
 #include "w_wad.h"
 #include "z_zone.h"
 
-#define DEFAULT_RAM 16 /* MiB */
+#if SIZEOF_VOID_P == 8
```

Comment:
1. There's no comment explaining why.
2. Why the complicated build system and preprocessor trickery? There's only one user of `DEFAULT_RAM`, you can just as well put this there and don't have to change the build system.

Reply:
I have to agree with @turol here. This change is too much to determine a single value that is used only once.

Reply:
Fixed in https://github.com/chocolate-doom/chocolate-doom/pull/1635/commits/4545a70dd14d941c0a570f12854e313ba73b483c